src/pages/_document.js

```diff
@@ -43,24 +44,30 @@ export default class MyDocument extends Document {
                 https://sozialhelden.matomo.cloud
                 https://cdn.matomo.cloud
                 http://cdn.matomo.cloud
+                https://api.mapbox.com
+                https://events.mapbox.com
 
+                http://localhost:4000
```

Comment:
This probably belongs in `REACT_APP_ALLOW_ADDITIONAL_DATA_URLS` in the dev env :)

Reply:
Oh yes, missed this while removing my local workarounds, will fix